canonical-web-and-design/jaas.ai
# Redirect logo

## Done

Redirect old logo path to the assets server to fix broken logo endpoint.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/static/img/logos/juju-logo.svg it should redirect to the assets server.
